problemsets-mat223/modules/module4.tex

```diff
@@ -138,7 +138,15 @@
 	Let $\vec a=\mat{1\\2}$, $\vec b=\mat{3\\3}$, $\vec c=\mat{2\\1}$, and $\vec v=\mat{3\\4}$. Which vector out of
 	$\vec a$, $\vec b$, and $\vec c$ has a direction closest to the direction of $\vec v$?
 
-	XXX Finish
+	To find the vector, we need to compute the angles $\alpha$, $\beta$ and $\gamma$ between vectors $\vec a$, $\vec b$, $\vec c$ and $\vec v$ respectively. 
```

Comment:
No you don't. You only need the cosine of the angles. Explain here why that's all you need.